CHANGELOG.md

```diff
@@ -3,6 +3,15 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.8.0] - Unreleased
```

Comment:
Add a "Changed" section with an item for MSRV bump here - that's the reason it's 0.8 and not 0.7.1, after all.

Reply:
✔️